### PR TITLE
Backport doc fixes to `release-v0.8.1`

### LINKED ISSUE #880: Backport docs fixes to release-v0.8.1

### PR DESCRIPTION
**Pull request by martriay**
Fixes #880.

### CODE REVIEW

File: docs/modules/ROOT/pages/presets.adoc [hunk 1]
```diff
@@ -1,15 +1,24 @@
...
-:erc721: xref:/api/erc721.adoc#ERC721[ERC721.cairo]
-:cairo-and-sierra: https://docs.starknet.io/documentation/architecture_and_concepts/Smart_Contracts/cairo-and-sierra/[Cairo and Sierra]
+TIP: For contract customization and combination of modules you can use {wizard}, our code-generation tool.
+
+
+## Available presets
+
+List of available presets and their corresponding {sierra-class-hashes}.
 
 NOTE: Class hashes were computed using {class-hash-cairo-version}.
```
[the remark this answers is not in the document]
Comment (martriay) [reply docs/modules/ROOT/pages/presets.adoc:23]: this needs to be addressed on `main` too then

[the remark this answers is not in the document]
Comment (andrew-fleming) [reply docs/modules/ROOT/pages/presets.adoc:23]: The second table header also (line 26) 😅

**Review by ericnordelo — APPROVED** [review]
LGTM!

**Review by andrew-fleming — APPROVED** [review]
LGTM!

**Pull request merged**